srv/salt/ceph/rgw/default.sls

```diff
@@ -7,15 +7,11 @@ install rgw:
     - name: ceph-radosgw
 
 {% for role in salt['pillar.get']('rgw_configurations', [ 'rgw' ]) %}
+
 start {{ role }}:
   service.running:
     - name: ceph-radosgw@{{ role + "." + grains['host'] }}
     - enable: True
 
-restart {{ role }}:
```

Comment:
similarly drop this section in default-ssl.sls as well

Reply:
good catch -- part of the next commit.